### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Gaurav-Dev1**

### CODE REVIEW

File: src/Auth/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const LoginButton = () => {
+  const { loginWithRedirect } = useAuth0();
+
+  return <button onClick={() => loginWithRedirect()}>Log In</button>;
```
Comment (SravaniKusuma) [inline src/Auth/Login.tsx:7]: use mui components and style home page

File: src/Components/Pages/BookDetailsPage/BookDetailsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Header from '../../organisms/Header/Header';
+import FooterComponent from '../../molecules/FooterBar/FooterComponent';
+import BookDescriptionLayout from '../../templates/BookDescription/BookDescriptionLayout';
+import BookDescription from '../../organisms/BookDescription/BookDescription';
+
+function BookDetailsPage() {
```
Comment (SravaniKusuma) [inline src/Components/Pages/BookDetailsPage/BookDetailsPage.tsx:7]: use arrow functions

File: src/Components/Pages/MyLibraryPage/MyLibraryPage.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import MyLibraryTemplate from '../../templates/MyLibrary/MyLibraryTemplate';
+import Tabs from '../../organisms/Tabs/Tabs';
+
+function MyLibraryPage() {
```
Comment (SravaniKusuma) [inline src/Components/Pages/MyLibraryPage/MyLibraryPage.tsx:5]: use arrow functions

Comment (SravaniKusuma) [inline src/Components/Pages/MyLibraryPage/MyLibraryPage.tsx:5]: use arrow functions

Comment (SravaniKusuma) [inline src/Components/Pages/MyLibraryPage/MyLibraryPage.tsx:5]: use arrow functions

File: src/Components/molecules/FooterBar/FooterComponent.tsx [hunk 4]
```diff
@@ -10,14 +10,14 @@ function FooterComponent() {
     return (
         
         <ThemeProvider theme={theme}>
-            <Box sx = {{padding: '24px 24px 24px 244px',width: '1440px',height: '370px',background: '#F1F6F4',position: 'absolute',bottom: '0px' }}>
+            <Box sx = {{padding: '24px 24px 24px 244px',width: '1440px',height: '370px',background: '#F1F6F4',bottom: '0px' }}>
```
Comment (SravaniKusuma) [inline src/Components/molecules/FooterBar/FooterComponent.tsx:13]: enable prettier format on save

File: src/Components/organisms/BookCard/BookCard.tsx [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+    left: 47.33,
+    fontSize: 14
+})
+
+let ButtonDiv = styled('div') ({
+    position: 'relative',
+    top: 10
+})
+
+let useStyles = makeStyles({
```
Comment (SravaniKusuma) [inline src/Components/organisms/BookCard/BookCard.tsx:63]: use const over let

File: src/Components/organisms/BookCard/BookCard.tsx [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+        if(bookInfo.status.isFinished) {
+            bookInfo.status.isFinished = false;
+        } else {
+            bookInfo.status.isFinished = true;
+        }
+        await api.put(`/library/${num}`, bookInfo)
+    }
+
+    const addToCurrentlyReading = async (num: number) => {
+        console.log('add to current library clicked')
```
Comment (SravaniKusuma) [inline src/Components/organisms/BookCard/BookCard.tsx:182]: remove logs

File: src/Auth/Login.tsx (hunk 1 above)
Comment (Gaurav-Dev1) [reply src/Auth/Login.tsx:7]: yeah, it will be done

**Pull request merged**